### PR TITLE
SERVICE : Create IFoodOrderService Interface And FoodOrderServie Class

### LINKED ISSUE #83: SERIVCE : Create IFoodOrderService Interface And FoodOrderService

### PR DESCRIPTION
**Pull request by ShohruhUzDev**
CLoses #83

### CODE REVIEW

File: BelissimoCloneWPF.Service/DTOs/FoodOrders/FoodOrderForUpdateDTO.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿namespace BelissimoCloneWPF.Service.DTOs.FoodOrders
+{
+    public class FoodOrderForUpdateDTO : FoodOrderForViewDTO
+    {
+    }
+}
```
Comment (MuhammadKasimov) [inline BelissimoCloneWPF.Service/DTOs/FoodOrders/FoodOrderForUpdateDTO.cs:6]: what is that?

File: BelissimoCloneWPF.Service/DTOs/FoodOrders/FoodOrderForViewDTO.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿namespace BelissimoCloneWPF.Service.DTOs.FoodOrders
+{
+    public class FoodOrderForViewDTO
+    {
+        public int Count { get; set; }
+        public int FoodId { get; set; }
+        public int BasketId { get; set; }
+
```
Comment (MuhammadKasimov) [inline BelissimoCloneWPF.Service/DTOs/FoodOrders/FoodOrderForViewDTO.cs:8]: remove

File: BelissimoCloneWPF.Service/Services/FoodOrders/FoodOrderService.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            this.mapper = mapper;
+        }
+
+        public async ValueTask<FoodOrderForViewDTO> CreateAsync(FoodOrderForCreationDTO foodOrderForCreationDTO)
+        {
+            var foodOrder = await foodOrderRepository.CreateAsync(mapper.Map<FoodOrder>(foodOrderForCreationDTO));
+            await foodOrderRepository.SaveChangesAsync();
+
+            return mapper.Map<FoodOrderForViewDTO>(foodOrder);
+
```
Comment (MuhammadKasimov) [inline BelissimoCloneWPF.Service/Services/FoodOrders/FoodOrderService.cs:30]: remove

**Review by MuhammadKasimov — CHANGES_REQUESTED** [review]

File: BelissimoCloneWPF.Service/DTOs/FoodOrders/FoodOrderForUpdateDTO.cs (hunk 1 above)
Comment (ShohruhUzDev) [reply BelissimoCloneWPF.Service/DTOs/FoodOrders/FoodOrderForUpdateDTO.cs:6]: it is DRY

Comment (MuhammadKasimov) [reply BelissimoCloneWPF.Service/DTOs/FoodOrders/FoodOrderForUpdateDTO.cs:6]: write it manually

Comment (MuhammadKasimov) [reply BelissimoCloneWPF.Service/DTOs/FoodOrders/FoodOrderForUpdateDTO.cs:6]: they are not the same man

**Pull request merged**